src/modules/display.js

```diff
@@ -1,38 +1,30 @@
 import { inputScore, name, score } from './variables.js';
+import getRaceScore from './carRace.js';
+import postScore from './post.js';
 
+// eslint-disable-next-line prefer-const
 let scoreArr = [];
 
-// Save To Local Directory
-const pushToLocal = () => {
-  localStorage.setItem('inputScore', JSON.stringify(scoreArr));
-};
-
-// To get recent scores and push to LocalStorage
 const addScore = () => {
   const user = {};
-  user.name = name.value;
+  user.user = name.value;
   user.score = score.value;
   scoreArr.push(user);
-  pushToLocal();
+  postScore(user);
+  name.value = '';
+  score.value = '';
 };
 
 // To render dynamically the recent scores
-const generateScore = () => {
-  scoreArr.forEach((user) => {
+const displayScore = async () => {
+  inputScore.innerHTML += '';
+  const scoreArr = await getRaceScore();
+  scoreArr.forEach((result) => {
```

Comment:
- [ ] Great job working on the functionality so far but  I think the display method is not working as expected since after submitting the name the the score and refresh the scores I can't see the new name and the new score so kindly fix the problem to make your project work as expected ✅🔥♻✨

# Let me show you here that I can't see the new name and score ♻🔥
- Before submitting
![image](https://user-images.githubusercontent.com/58553711/209251994-00e5663a-e932-41a1-94f8-6afa880d78b7.png)
- After
![image](https://user-images.githubusercontent.com/58553711/209252079-f716fb3c-6abe-44c9-813e-af44245796b4.png)
![image](https://user-images.githubusercontent.com/58553711/209252104-8acb221d-50c5-4799-b224-ad2554f3b4af.png)